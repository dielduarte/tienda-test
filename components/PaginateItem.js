import styled from 'styled-components';

const PaginateItem = styled.span`
  display: inline-block;
  text-align: center;
  margin-right: 2px;
  padding-top: 2px;
  color: ${props => props.theme.white};
  font-size: 14px;
  background: ${props => props.theme.gray};
  width: 20px;
  height: 20px;

  &:hover {
    cursor: pointer;
  }
`;

export default PaginateItem;
