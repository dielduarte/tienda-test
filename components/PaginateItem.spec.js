import { shallow } from 'enzyme';
import 'jest-styled-components';

import PaginateItem from './PaginateItem';

describe('PaginateItem', () => {
  it('PaginateItem', () => {
    const wrapper = shallow(<PaginateItem>1</PaginateItem>);
    expect(wrapper).toMatchSnapshot();
  });
});
