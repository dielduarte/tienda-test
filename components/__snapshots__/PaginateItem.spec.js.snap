// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginateItem PaginateItem 1`] = `
.c0 {
  display: inline-block;
  text-align: center;
  margin-right: 2px;
  padding-top: 2px;
  font-size: 14px;
  width: 20px;
  height: 20px;
}

.c0:hover {
  cursor: pointer;
}

<span
  className="c0"
>
  1
</span>
`;
